sanic/blueprints.py

```diff
@@ -54,7 +55,10 @@ def chain(nested):
                     yield i
         bps = []
         for bp in chain(blueprints):
+            if bp.url_prefix is None:
+                bp.url_prefix = ''
             bp.url_prefix = url_prefix + bp.url_prefix
+            bp.url_prefix = os.path.normpath(bp.url_prefix)
```

Comment:
I don't think it should call normpath here.

Reply:
@yunstanford there's a discussion regarding this on #1325, @chenjr0719 just needs to update the PR afaik.

Reply:
@vltr @yunstanford I just remove `normpath` to make this PR simple and focus on the original issue about default value of `url_prefix`.